File: EvenOdd.cpp
#include <iostream>

using namespace std;

void evenodd(int n)
{
    if(n%2==0)
    {
        cout << n << " is a even number" << endl;  
    }
    else
    {
        cout << n << " is a even number" << endl;  
    }
}


int main() {
 
    int number;
    cout << "Enter a number : " ;
    cin >> number;
    evenodd(number);

    return 0;

}

File: EvenOdd.java
import java.lang.System;
import java.util.Scanner;

public class EvenOdd {
    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            System.out.print("Enter an integer : ");
            int num = input.nextInt();
            if (num % 2 == 0) {
                System.out.println(num + " is a Even number");
            } else {
                System.out.println(num + " is a Odd number");
            }
        }
    }
}
